src/math/p5.Vector.js

```diff
@@ -1149,54 +1193,60 @@ p5.Vector = class {
   }
 
   /**
- * Calculates the Euclidean distance between two points (considering a
- * point as a vector object).
- * If you are looking to calculate distance between 2 points see <a href="#/p5/dist">dist()</a>
+ * Returns the distance between the tips of two vectors extending from the
```

Comment:
It might be worth keeping the point interpretation from before, for me at least there are less mental steps involved in picturing what's going on. The vector interpretation is also helpful, but maybe as a secondary explanation?

Reply:
Agreed, let me know how this draft reads.